doc/source/docs.rst

```diff
@@ -18,7 +18,6 @@ scientific installations.
    devel/index
    sep/index.rst
    Glossary <glossary>
-   other_versions
```

Comment:
Even if the other versions do not get automatically built, I think that it is interesting keeping a pointer to some selected older versions of the docs (ideally to the officially released versions, but if not, at least to the v2.x) so that someone using an older version could see the docs for their version. 

I do not see this as blocking or even worth investing too much time, but if it can be kept without much efort, I would keep it. If not, maybe a note somewhere indicating how an user can build the older docs from her own copy of the source could be added.

Reply:
For the moment I just added a [guide](https://reszelaz.github.io/sardana-doc/other_versions.html) on how to build the docs for other versions.

Note that until now we did not really have other versions of the docs. The [page](https://sardana-controls.org/other_versions.html) with other versions just contains other branches uploaded to the upstream repo. No one never complained that the other versions were missing. Furthermore, before holidays the develop branch was already migrated to python 3. Since then also no one complained about the last python 2 version of the docs. That's why, IMHO, for the moment I would continue as we are. As an extra, with this PR we will have instructions on how to build other versions of the docs. If there is a strong interest from the Community for online other versions we can reconsider adding them.

Reply:
ok. I just added a small change to be more explicit about the command that needs to be executed for building the docs.

Also, as aside note: accepting this PR has implications on the debian packaging: it makes [this patch](https://salsa.debian.org/science-team/sardana/blob/master/debian/patches/0001-Remove-embedded-script-avoid-privacy-breach-lintian-.patch) unnecessary.